client/src/pages/Review/Review.tsx

```diff
@@ -163,15 +163,16 @@ const Review = () => {
     );
   };
 
+  const isMobile = () => ["xs", "sm"].includes(breakpoint);
```

Comment:
I really wrote this code last night, merged the branch but forgot to push the changes 🤦‍♂️